test-apps/display-performance-test-app/src/frontend/TestRunner.ts

```diff
@@ -1028,51 +1037,50 @@ function getRenderOpts(curRenderOpts: RenderSystem.Options): string {
           }
         }
         break;
-      case "preserveShaderSourceCode":
-        if (value) optString += "+shadeSrc";
-        break;
+      }
       case "displaySolarShadows":
-        if (!value) optString += "-solShd";
+        if (!curRenderOpts[key]) optString += "-solShd";
         break;
-      case "logarithmicZBuffer":
-        if (value) optString += "+logZBuf";
+      case "logarithmicDepthBuffer":
+        if (!curRenderOpts[key]) optString += "-logZBuf";
         break;
       case "useWebGL2":
-        if (value) optString += "+webGL2";
+        if (!curRenderOpts[key]) optString += "-webGL2";
         break;
-      case "antialiasSamples":
-        if (value > 1) optString += `+aa${value as number}`;
+      case "antialiasSamples": {
+        const value = curRenderOpts[key];
+        if (value && value > 1) optString += `+aa${value}`;
         break;
-      default:
-        if (value) optString += `+${key}`;
+      }
     }
   }
+
   return optString;
 }
 
 function getTileProps(curTileProps: TileAdmin.Props): string {
   let tilePropsStr = "";
-  for (const [key, value] of Object.entries(curTileProps)) {
-    switch (key) {
-      case "elideEmptyChildContentRequests":
-        if (value) tilePropsStr += "+elide";
-        break;
-      case "enableInstancing":
-        if (value) tilePropsStr += "+inst";
-        break;
-      case "maxActiveRequests":
-        if (value !== 10) tilePropsStr += `+max${value}`;
-        break;
-      case "retryInterval":
-        if (value) tilePropsStr += `+retry${value}`;
-        break;
-      case "disableMagnification":
-        if (value) tilePropsStr += "-mag";
-        break;
-      default:
-        if (value) tilePropsStr += `+${key}`;
-    }
-  }
+  if (!curTileProps.enableInstancing)
```

Comment:
Many of these have no effect on the image. Other properties that weren't being included do affect the image. I changed some to only add to the test name if they differ from their default values.